.circleci/config.yml

```diff
@@ -58,5 +58,5 @@ jobs:
              if [ "${CIRCLE_BRANCH}" == "dev" ]; then
                pip install scikit-ci-addons
                ci_addons publish_github_release NeurodataWithoutBorders/pynwb \
-                 --prerelease-packages "dist/*" --exit-success-if-missing-token
+                 --prerelease-packages "dist/*" --exit-success-if-missing-token --re-upload
```

Comment:
An other approach would be:

```
--prerelease-packages "dist/*" \
--prerelease-packages-clear-pattern "*.*" \
--prerelease-packages-keep-pattern "*.g<COMMIT_DATE>*.*" \
--exit-success-if-missing-token
```

that said, this wouldn't allow us to collect the package generated after tagging the repo.

If using `--re-upload` works, let's go with it.